scripts/websockClient.js

```diff
@@ -0,0 +1,16 @@
+const { WebSocketClient } = require('..');
+const ws = require('ws');
+const args = process.argv.slice(2);
+
+// Configuration parameters
+const HOST = args[0];
+const PORT = args[1];
+
+let data="";
+
+const wsc = new WebSocketClient({ host: HOST, port: PORT, WebSocket: ws });
+
+//Optional listener to retrieve data
+wsc.addMessageListener(({ userName, message }) => {
+  data=`${userName}: ${message}`;
```

Comment:
Absolutely, what I intend is to keep the normal cli chat prompt as it is and yet provide an interface for the user to directly use the data received maybe for some other processing. I think that latter is like an optional feature for the user rather than a necessity but as we planned this thing as a chat service I guess the default console display is kind of the default behavior. We can surely add in options to let the user enable the console log or just completely get all the data feed. Let me know what you think.